source/features/github-actions-indicators.tsx

```diff
@@ -43,18 +69,40 @@ const getWorkflowsDetails = cache.function('workflows', async (): Promise<Record
 		}
 	`);
 
-	const workflows = workflowFiles?.entries ?? [];
+	const workflows: [any] = workflowFiles?.entries ?? [];
+
+	const result: Record<string, string> = {};
+	for (const workflow of workflows) {
+		result[workflow.name] = workflow.object.text;
+	}
+
+	return result;
+};
+
+const getWorkflowsDetails = cache.function('workflows', async (): Promise<Record<string, WorkflowDetails> | false> => {
+	const workflows = await getWorkflows();
+
 	if (workflows.length === 0) {
 		return false;
 	}
 
+	const workflowYamls = await getWorkflowYamls();
```

Comment:
Since we can't avoid making two calls, I'd make these two calls in parallel. I'd also add a comment above the v3 call linking back to this PR, so that we know why it's not using v4


I agree that we should use the new `workflows` list as the source of truth for the workflow list. I think `getWorkflowYamls` currently returns any file in the folder at the moment

Reply:
To resolve both queries concurrently, `getWorkflowYamls` cannot return "validated files".
So I'll just change the function name to `getFilesInWorkflowPath`